### ISSUE TITLE
Check for '.' in the bucket name and produce an error message if needed

### ISSUE THREAD

**Issue by dack-fe** [fpga-open-speech-tools/autogen_webapp#7]
The web app is not able to parse `.` in the bucket name correctly, even though this is a valid [S3 bucket name](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html)

![image](https://user-images.githubusercontent.com/53094583/103317073-b0f43700-49e7-11eb-96dd-ea4c0fb1f737.png)

**Comment by tvannoy** [comment]
As mentioned in the email thread, using `.` in bucket names is not suggested. From the [AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html):
> If you include dots in a bucket's name, you can't use virtual-host-style addressing over HTTPS, unless you perform your own certificate validation. This is because the security certificates used for virtual hosting of buckets don't work for buckets with dots in their names. 

Judging by the screen capture in this issue, we are using virtual-host-style addressing over https, which is why we are getting a certificate error. We could technically use buckets with `.` in their names if we used [path-style requests](https://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html), but path-style requests are being deprecated.

It looks like we basically need to not use `.` in bucket names.

**Comment by dack-fe** [comment]
@tvannoy I agree the best practice is to no use `.` in bucket names. I think we should update our documentation to specify that we don't support `.` and have the web app provide an error message if it detects `.` in the bucket name.
